src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -59,20 +59,20 @@ def show_subscription(cmd, subscription=None, show_auth_for_sdk=None):
     print(json.dumps(profile.get_sp_auth_info(subscription), indent=2))
 
 
-def get_access_token(cmd, subscription=None, resource=None, resource_type=None):
-    '''
+def get_access_token(cmd, subscription=None, resource=None, resource_type=None, tenant=None):
+    """
     get AAD token to access to a specified resource
     :param resource: Azure resource endpoints. Default to Azure Resource Manager
     :param resource-type: Name of Azure resource endpoints. Can be used instead of resource.
```

Comment:
please also add tenent in comment

Reply:
The doc string is overridden by https://github.com/Azure/azure-cli/blob/fbfe69172600936798387e2b5c9c1c9887bb38f4/src/azure-cli/azure/cli/command_modules/profile/__init__.py#L78
I'd prefer to omit here to eliminate duplication.